base/complex.jl

```diff
@@ -7,6 +7,8 @@ Complex number type with real and imaginary part of type `T`.
 
 `ComplexF16`, `ComplexF32` and `ComplexF64` are aliases for
 `Complex{Float16}`, `Complex{Float32}` and `Complex{Float64}` respectively.
+
+See also: [`Real`](@ref), [`complex`](@ref), [`real`](@ref).
```

Comment:
Seems like we weren't typically using a `:`, except this PR (which is inconsistent about it)? I think I'd suggest removing them all from this PR. Other opinions?

Reply:
I'm not completely consistent, but my thinking was that some (esp. of the old ones) are more part of the discussion, "See also X which does Y, and Z.", so should be punctuated as sentences. While some are just a list of links, "See also: X, Y, Z."

Reply:
Yes, I think removing makes it more consistent for cases where a sentence is better than a simple list, and thus is overall better to use always. Let's see how the voting goes: https://github.com/JuliaLang/julia/pull/40401